CHANGELOG.md

```diff
@@ -1,7 +1,15 @@
 # Change Log
 ## [master](https://github.com/dallinger/dallinger/tree/master) (xxxx-xx-xx)
 
-## [v8.1.0](https://github.com/dallinger/dallinger/tree/v8.1.0) (2021-03-25)
+## [v9.0.0](https://github.com/dallinger/dallinger/tree/v9.0.0) (2022-05-19)
+- Infrastructure: Drop support for Python 3.7
+- Infrastructure: Update versions of various dependencies, including those which had been impossible while retaining Python 3.7 compatibility, e.g. Flask to v2.x, Jinja2 to v3.x, numpy to 1.22.x, pandas to v1.4.x, click to 8.x and Sphinx to 4.5.x
+- Feature: Add `clock` support in Docker
```

Comment:
I would maybe call this a bugfix

Reply:
There is a problem which I am unable to resolve with generating the constraints for the demos without releasing the new version first. This is because the former version has the `Flask<2.0` constraint:
```
There are incompatible versions in the resolved dependencies:
  flask==2.1.2 (from -c ../../../../dev-requirements.txt (line 143))
  Flask<2.0 (from dallinger==8.1.0->-r requirements.txt (line 1))
```

Reply:
Hmm, that's annoying

Reply:
@jessesnyder , maybe you have an idea how to resolve this?